apps/server/test/lexical/server/code_intelligence/completion/builder_test.exs

```diff
@@ -34,41 +35,35 @@ defmodule Lexical.Server.CodeIntelligence.Completion.BuilderTest do
     :ok
   end
 
-  describe "boosting" do
-    test "default boost sorts things first" do
-      alpha_first = item("a")
-      alpha_last = "z" |> item() |> boost()
-
-      assert [^alpha_last, ^alpha_first] = sort_items([alpha_first, alpha_last])
+  describe "sort scopes" do
+    test "scope order follows variable -> local -> remote -> global -> auto -> default" do
+      i = set_sort_scope(item("g"), SortScope.module())
+      ii = set_sort_scope(item("f"), SortScope.variable())
+      iii = set_sort_scope(item("e"), SortScope.local())
+      iv = set_sort_scope(item("d"), SortScope.remote())
+      v = set_sort_scope(item("c"), SortScope.global())
+      vi = set_sort_scope(item("b"), SortScope.auto())
+      vii = set_sort_scope(item("a"), SortScope.default())
+
+      assert [^i, ^ii, ^iii, ^iv, ^v, ^vi, ^vii] = sort_items([vii, vi, v, iv, iii, ii, i])
```

Comment:
I can't tell if I like this or not, but it's exceedingly concise and clever.

Reply:
I debated keeping the `alpha_first`/`alpha_second` naming scheme, but I felt this looked nicer, particularly keeping the assert on a single line.

Reply:
Like i said, clever, though it might be difficult for those not exposed to roman numerals (@scottming ?) to understand.